pages/HOME.md

```diff
@@ -1,69 +1,88 @@
-  * [/r/demo/users](https://test3.gno.land/r/demo/users)
-* p/demo
-  * [/p/demo/avl](https://test3.gno.land/p/demo/avl)
-  * [/p/demo/blog](https://test3.gno.land/p/demo/blog)
-  * [/p/demo/flow](https://test3.gno.land/p/demo/flow)
-  * [/p/demo/gnode](https://test3.gno.land/p/demo/gnode)
-  * [/p/demo/grc/exts](https://test3.gno.land/p/demo/grc/exts)
-  * [/p/demo/grc/grc20](https://test3.gno.land/p/demo/grc/grc20)
-  * [/p/demo/grc/grc721](https://test3.gno.land/p/demo/grc/grc721)
-
-# Other Testnets
-
-   * **[staging.gno.land](https://staging.gno.land) (wiped every commit to master)**
-   * _[test3.gno.land](https://test3.gno.land) (latest)_
-   * _[test2.gno.land](https://test2.gno.land) (archive)_
-   * _[test1.gno.land](https://test1.gno.land) (archive)_
-
-**This is a testnet.**
-Package names are not guaranteed to be available for production.
-
-# Social
-
-Check out our [community projects](https://github.com/gnolang/awesome-gno).
-
-Official channel: [Discord](https://discord.gg/S8nKUqwkPn)<br />
-Other channels: [Telegram](https://t.me/gnoland) [Twitter](https://twitter.com/_gnoland) [Youtube](https://www.youtube.com/@_gnoland)
-
-<!-- mailchimp -->
-<div id="mc_embed_signup">
-<form action="https://land.us18.list-manage.com/subscribe/post?u=8befe3303cf82796d2c1a1aff&amp;id=271812000b&amp;f_id=009170e7f0" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_self">
-  <label for="mce-EMAIL">Subscribe by email:</label>
-  <div id="mc_embed_signup_scroll">
-  	<div class="mc-field-group">
-  		<input type="email" value="" name="EMAIL" class="required email" id="mce-EMAIL" placeholder="Type your email here" required>
-  		<input type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe" class="button">
-  		<span id="mce-EMAIL-HELPERTEXT" class="helper_text"></span>
-  	</div>
-  	<div id="mce-responses" class="clear">
-  		<div class="response" id="mce-error-response" style="display:none"></div>
-  		<div class="response" id="mce-success-response" style="display:none"></div>
-  	</div>
-  	<div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_8befe3303cf82796d2c1a1aff_271812000b" tabindex="-1" value=""></div>
-  </div>
-</form>
-</div>
-<!-- /mailchimp -->
+# Welcome to Gno.land
+
+---
+
+## An interpretation of the Golang (Go) programming language for advanced developers and intrepid pioneers to build succinct, composable smart contracts for social coordination.
+
+If you’re concerned about information censorship and want to contribute to the #GnoWorldOrder, follow our socials to find out how.
+
+Gno.land is in building mode. If you want to help lay the foundations of a fairer and freer world through innovative ideas and exceptional code, join us today.
+
+<div class="stack">
+
+[About Gno.land](/about)
+[Explore Github repo](https://github.com/gnolang)
+
+</div><!-- end stack-->
```

Comment:
```suggestion
<div class="stack">
[About Gno.land](/about)
[Explore Github repo](https://github.com/gnolang)
</div><!-- end stack-->
```

grouped by goal looks better to me

Reply:
@moul Yeah, but it's not possible. As I told you in the previous Gno [PR](https://github.com/gnolang/gno/pull/818), HTML in markdown is a nightmare because of MD writing convention:

> But I agree, like I said in a meeting we had earlier, using div in markdown is not very user friendly as we don't have indentation and we must follow line break + space convention that are not common in HTML... That's why our futur component lib should help here.

If I do that, the inner markdown will not be rendered as markdown but as HTML

<img width="1107" alt="Capture d’écran 2023-06-19 à 19 35 57" src="https://github.com/gnolang/www.gno.land/assets/7622257/585eb58f-e598-48d0-bd95-bdad521534ab">

Reply:
ok, so yeah, maybe not the best solution for long term; let's keep this for now.

I think we should do something more markdownish, like https://marp.app/.

<img width="889" alt="CleanShot 2023-06-22 at 18 04 12@2x" src="https://github.com/gnolang/www.gno.land/assets/94029/81d421f1-e0e6-4656-95e8-989feaedee96">